src/api/endpoints/healthcheck_email.py

```diff
@@ -0,0 +1,49 @@
+"""Module to test Notifications being sent out."""
+
+from flask import request, current_app
+from flask_restplus import Resource
+
+from api.utils.auth import token_required
+from api.utils.helpers import response_builder
+from api.utils.notifications.email_notices import send_email
+
+
+class NotificationsAPI(Resource):
+    """
+    Notifications Resource.
+
+    Use to confirm that Notifications are actually being sent out.
+    """
+
+    decorators = [token_required]
+
+    @classmethod
+    def post(cls):
+        """Trigger send of email to predetermined email address."""
+        payload = request.get_json(silent=True)
+        if not payload:
+            return response_builder(dict(
+                                    message="Data for sending must "
+                                            "be provided.",
+                                    status="fail",
+                                    ), 400)
+
+        recipient = payload.get("recipient")
```

Comment:
Maybe a marshmallow email validation here?

Reply:
On it.

Reply:
It's registered with the app with no restrictions on who can use it. Essentially all Andelans can use it. Imagine an Andelan with ill intent.

Reply:
makes sense